dev/dialects/db2/11.5/start.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
What's more difficult to merge? Are you suggesting back-porting to other versions?

While it may seem like more noise, it keeps convention -- no other place are there dialect folders (e.g., `/db2`, `/mysql`, `/postgres`) that aren't under a `/dialects` folder:

- /src/dialects/*
- /lib/dialects/*
- /test/integration/dialects/*
- /test/unit/dialects/*

The other places may have good reason to have that pattern, but would `/dev` be the _only_ one to break that convention? The directory should hardly be used, so consistency seems more important.

Currently the scripts are greping `start.sh` to get the dialect name.  What would be better is to iterate over the directory structure and not rely on grep or linux shell commands.

Reply:
I think she is mostly referring to your other open PRs but I'm fine with doing it either way

Reply:
The reasoning is that changes with a large diff impact that don't have a purpose should be avoided to prevent unnecessary collisions with other PRs that may be touching the same files, as well as reducing diff size for review

In the examples you listed, they all mimic `src`, but there is no convention about a `dialects` directory. `lib` is the built version of `src`, and tests try to mimic the structure of src so tests are easier to find.

That being said, changing `start` to making it list the contents of the folder instead of a grep would be better, and is reason enough to introduce this `/dialect` folder